src/main/java/coffee/client/feature/module/impl/combat/AimAssist.java

```diff
@@ -163,13 +190,102 @@ public String getContext() {
+                else if(aimMode.getValue() == AimMode.Custom)
+                {
+                    Rotation py = Rotations.getPitchYaw(le.getPos().add(0, (le.getHeight() / 100) * aimHeight.getValue(), 0));
+                    Objects.requireNonNull(CoffeeMain.client.player).setPitch(py.getPitch());
+                    CoffeeMain.client.player.setYaw(py.getYaw());
+                }
+            }
         }
+        else if (!aimRandom.getValue())
+        {
+            if (!aimInstant.getValue()) {
+                if(aimMode.getValue() == AimMode.Top)
+                {
+                    Rotations.lookAtPositionSmooth(le.getPos().add(0, (le.getHeight() / 1.3d), 0), (float) (laziness.getValue() + 0));
+                }
+                if(aimMode.getValue() == AimMode.Middle)
+                {
+                    Rotations.lookAtPositionSmooth(le.getPos().add(0, (le.getHeight() / 2d), 0), (float) (laziness.getValue() + 0));
+                }
+                else if(aimMode.getValue() == AimMode.Bottom)
+                {
+                    Rotations.lookAtPositionSmooth(le.getPos().add(0, (le.getHeight() / 4d), 0), (float) (laziness.getValue() + 0));
+                }
+                else if(aimMode.getValue() == AimMode.Bottom)
+                {
+                    Rotations.lookAtPositionSmooth(le.getPos().add(0, (le.getHeight() / 100) * aimHeight.getValue(), 0), (float) (laziness.getValue() + 0));
+                }
+            } else {
+                if(aimMode.getValue() == AimMode.Top)
+                {
+                    Rotation py = Rotations.getPitchYaw(le.getPos().add(0, (le.getHeight() / 1.3d), 0));
+                    Objects.requireNonNull(CoffeeMain.client.player).setPitch(py.getPitch());
+                    CoffeeMain.client.player.setYaw(py.getYaw());
+                }
+                else if(aimMode.getValue() == AimMode.Middle)
+                {
+                    Rotation py = Rotations.getPitchYaw(le.getPos().add(0, le.getHeight() / 2d, 0));
+                    Objects.requireNonNull(CoffeeMain.client.player).setPitch(py.getPitch());
+                    CoffeeMain.client.player.setYaw(py.getYaw());
+                }
+                else if(aimMode.getValue() == AimMode.Bottom)
+                {
+                    Rotation py = Rotations.getPitchYaw(le.getPos().add(0, le.getHeight() / 4d, 0));
+                    Objects.requireNonNull(CoffeeMain.client.player).setPitch(py.getPitch());
+                    CoffeeMain.client.player.setYaw(py.getYaw());
+                }
+                else if(aimMode.getValue() == AimMode.Custom)
+                {
+                    Rotation py = Rotations.getPitchYaw(le.getPos().add(0, (le.getHeight() / 100) * aimHeight.getValue(), 0));
+                    Objects.requireNonNull(CoffeeMain.client.player).setPitch(py.getPitch());
+                    CoffeeMain.client.player.setYaw(py.getYaw());
+                }
+            }
+        }
```

Comment:
same code but with a different variable, you can just use the modifiedLaziness and only change it when randomness is enabled